cmd/utils/flags.go

```diff
@@ -757,7 +757,7 @@ var (
 	// fetcher flag to set arrival timeout
 	TxArrivalWaitFlag = cli.IntFlag{
 		Name:  "txarrivalwait",
-		Usage: "Maximum number of milliseconds to wait for a transaction before requesting it (defaults to 100ms)",
+		Usage: "Maximum number of milliseconds to wait for a transaction before requesting it (defaults to 500ms)",
```

Comment:
I believe not it is not about milliseconds. Could you change comments accordingly?

Reply:
this is odd - I had already updated that line. I'm not sure why it's pulling up an older commit's text..

https://github.com/thogard785/bor/blob/d56370b9101717d7b48c5862416a8069df949174/cmd/utils/flags.go#L761

GitHub is out to get me I swear lol.